MusicTaste/MusicTaste/Cells/User.m

```diff
@@ -30,13 +31,15 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
     //Encode properties such as name and the profile picture
     [encoder encodeObject:self.name forKey:@"name"];
     [encoder encodeObject:self.profilePicture forKey:@"image"];
+    [encoder encodeObject:self.profilePicture forKey:@"artists"];
 }
 
 - (id)initWithCoder:(NSCoder *)decoder {
     if((self = [super init])) {
         //decode name and image
         self.name = [decoder decodeObjectForKey:@"name"];
         self.profilePicture = [decoder decodeObjectForKey:@"image"];
+        self.profilePicture = [decoder decodeObjectForKey:@"artists"];
```

Comment:
When we talked yesterday in our 1-1 we came to the conclusion that encoding/decoding might not actually be doing anything; was that conclusion wrong? Seems important to understand what is happening here, and to remove it if it's not necessary.

Reply:
Will be deleting Use in the next iteration.